src/commands/build/wranglerjs/mod.rs

```diff
@@ -58,6 +58,8 @@ pub fn run_build_and_watch(target: &Target, tx: Option<Sender<()>>) -> Result<()
     let (mut command, temp_file, bundle) = setup_build(target)?;
     command.arg("--watch=1");
 
+    let is_site = target.site.clone();
```

Comment:
This should be maybe_site I think

Reply:
why maybe site? i think is_site says if it is a site or not, the maybe will be very tough to understand later. if target.site exists, it's a site. am i missing something perhaps?